.travis.yml

```diff
@@ -1,14 +1,14 @@
+sudo: required
```

Comment:
> I checked the real "container" env with sudo: false, but while it
still offers us to use "Ubuntu 14.04" image, jruby in this case gives
us jruby 1.7.26. So in the end I just fixed the commit message.

Now that I think about it, that's another instance of the same problem fixed in https://github.com/Locale/localeapp/pull/213/commits/962c184f27c37a13542334efc0f99ec08db5c5e2, so to fix it I guess you can just specify `jruby-9.1.6.0` just like you do for the other rubies.